docs/howto/debugging.md

```diff
@@ -81,6 +82,23 @@ See also [in the "Troubleshooting" section below](#debug-remotely-never-loads).
 [chrome-devtools-device]: https://facebook.github.io/react-native/docs/debugging.html#debugging-on-a-device-with-chrome-developer-tools
 
 
+<div id="react-devtools">
+
+## React Devtools
+
+The standalone version of React DevTools, which is an electron app, can be used
+to debug  the react-native component hierarchy. Refer to
+[the docs](https://facebook.github.io/react-native/docs/debugging.html#react-sdeveloper-tools)
```

Comment:
An important step when writing documentation with links in it: test out the links to make sure you don't have a typo in them :wink: